src/migration/1727578711381-VordrUser.ts

```diff
@@ -0,0 +1,90 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class VordrUser1727578711381 implements MigrationInterface {
+  name = 'VordrUser1727578711381'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE INDEX "IDX_post_flags_vordr" ON post (((flags->'vordr')::boolean))`);
+
+    await queryRunner.query(`CREATE OR REPLACE FUNCTION vordr_update_flags()
+RETURNS TRIGGER AS $$
+BEGIN
+  IF NEW.flags @> '{"vordr": true}' THEN
+    -- If vordr on user is set to true
+    -- Set banned and vordr to true in post table, and set vordr to true in comment table
+    UPDATE
+      post
+    SET
+      "banned" = TRUE,
+      "flags" = jsonb_set(jsonb_set(flags,
+        '{vordr}', 'true', true),         -- set vordr to true
+        '{banned}', 'true', true)         -- set banned to true
+    WHERE
+      ("authorId" = NEW.id OR "scoutId" = NEW.id);
+
+    UPDATE
+      comment
+    SET
+      "flags" = jsonb_set(flags, '{vordr}', 'true', true)
+    WHERE
+      "userId" = NEW.id;
+
+  -- If vordr on user is set to false, and vordr on post is true
+  -- Set banned and vordr to false in post table, and set vordr to false in comment table
+  ELSIF NEW.flags @> '{"vordr": false}' THEN
```

Comment:
Just interested, currently we rarely/never set vordr back to false?

Reply:
Not sure if this is covered again? https://github.com/dailydotdev/daily-api/pull/2289#discussion_r1780797883, if possible at test case for it

Reply:
It's covered in the test `should update all posts the user has made when the vordr flag is set to false`
But I could add an explicit test for it

Reply:
Added explicit test